com.tomjwatson.Emote.yml

```diff
@@ -0,0 +1,68 @@
+app-id: com.tomjwatson.Emote
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: emote
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=xdg-config/autostart:create
```

Comment:
You should use the portals for that.

Reply:
This is why the build failed. Note that this is code that needs to be written. not "something in the manifest"

Reply:
No problem if there is a bit of code to be written, but is there any example or documentation about how we can enable a flatpak package to start automatically on boot in the background using the portal API?

Reply:
Found something that works:

```python
import dbus
bus = dbus.SessionBus()
obj = bus.get_object("org.freedesktop.portal.Desktop", "/org/freedesktop/portal/desktop")
inter = dbus.Interface(obj, "org.freedesktop.portal.Background")
res = inter.RequestBackground('', {
    'reason': 'Emote autostart',
    'autostart': True, 'background': True,
    'commandline': dbus.Array(['emote'])
})
```

Will push the fix to try a new build asap

Reply:
I just pushed the fix without the filesystem permission and using the portal Background API, thanks for the recommendation @hfiguiere